### DISCUSSION TITLE
Custom rendition names

### DISCUSSION THREAD

**Discussion by nwang92** [muxinc/media-chrome#927] [Q&A] [answered]
I have some HLS playlists without `RESOLUTION` set, and when using hls-video element + media-chrome to support rendition selection, I get the following:
<img width="113" alt="Screenshot 2024-06-20 at 6 48 31 PM" src="https://github.com/muxinc/media-chrome/assets/6201822/33de3e8f-03d1-495b-9614-0f42ceabc992">

I'm guessing this is occurring because each rendition's height is set to 0. Is there a way to override the displayed values here?

**Comment by luwes** [comment, marked as answer]
yes, this is possible.

```js
      const renditionMenus = document.querySelectorAll('media-rendition-menu');

      for (const renditionMenu of renditionMenus) {
        renditionMenu.formatMenuItemText = (text, rendition) => {
          console.log(text, rendition);
          return text.replace('1080p', '1080p (HD)');
        }
      }
```

see an example here https://media-chrome.mux.dev/examples/vanilla/control-elements/media-rendition-menu.html

**Comment by nwang92** [comment]
Works for me, thanks!
